websites/P/Perplexity/presence.ts

```diff
@@ -0,0 +1,85 @@
+const presence = new Presence({
+		clientId: "893871116945350726",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	threadExportRegex = /([a-f\d]{8}(-[a-f\d]{4}){3}-[a-f\d]{12})/i;
+let recentSearchQuery: string = null;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/gdCcljx.png",
+			startTimestamp: browsingTimestamp,
+		},
+		{ pathname } = document.location,
+		{ href } = document.location,
```

Comment:
This looks cleaner and works the same
```suggestion
		{ href, pathname } = document.location,
```